pandas/tslib.pyx

```diff
@@ -1098,6 +1102,10 @@ cdef class _Timestamp(datetime):
         out = get_start_end_field(np.array([self.value], dtype=np.int64), field, freqstr, month_kw)
         return out[0]
 
+    cpdef _get_date_name_field(self, field):
```

Comment:
Yes, we can just do:
`return ["Monday","Tue...][self.weekday()]`
in the property, no?
Although, I did find f_weekday in LocaleTime,.. not sure how to access that...